[2022-08-03 02:00:00,421] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.download_dataset_task scheduled__2022-08-02T00:00:00+00:00 [queued]>
[2022-08-03 02:00:00,439] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.download_dataset_task scheduled__2022-08-02T00:00:00+00:00 [queued]>
[2022-08-03 02:00:00,440] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 02:00:00,441] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-08-03 02:00:00,441] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 02:00:00,463] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2022-08-02 00:00:00+00:00
[2022-08-03 02:00:00,471] {standard_task_runner.py:52} INFO - Started process 63 to run task
[2022-08-03 02:00:00,474] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'download_dataset_task', 'scheduled__2022-08-02T00:00:00+00:00', '--job-id', '96', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmp176oi8xv', '--error-file', '/tmp/tmp4vyh50hd']
[2022-08-03 02:00:00,476] {standard_task_runner.py:77} INFO - Job 96: Subtask download_dataset_task
[2022-08-03 02:00:00,545] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag.download_dataset_task scheduled__2022-08-02T00:00:00+00:00 [running]> on host ce750bf29e22
[2022-08-03 02:00:00,623] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2022-08-02T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-02T00:00:00+00:00
[2022-08-03 02:00:00,624] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-03 02:00:00,625] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://storage.cloud.google.com/dtc_data_lake_de-bootcamp-357803/covid-cases.csv > /opt/***/covid-cases.csv']
[2022-08-03 02:00:00,640] {subprocess.py:85} INFO - Output:
[2022-08-03 02:00:00,727] {subprocess.py:93} INFO - Command exited with return code 0
[2022-08-03 02:00:00,753] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=download_dataset_task, execution_date=20220802T000000, start_date=20220803T020000, end_date=20220803T020000
[2022-08-03 02:00:00,814] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-08-03 02:00:00,897] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-08-03 04:05:37,653] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.download_dataset_task scheduled__2022-08-02T00:00:00+00:00 [queued]>
[2022-08-03 04:05:37,670] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.download_dataset_task scheduled__2022-08-02T00:00:00+00:00 [queued]>
[2022-08-03 04:05:37,671] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 04:05:37,671] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-08-03 04:05:37,672] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 04:05:37,687] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2022-08-02 00:00:00+00:00
[2022-08-03 04:05:37,696] {standard_task_runner.py:52} INFO - Started process 82 to run task
[2022-08-03 04:05:37,701] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'download_dataset_task', 'scheduled__2022-08-02T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmpnbrbzyil', '--error-file', '/tmp/tmpannumw6g']
[2022-08-03 04:05:37,703] {standard_task_runner.py:77} INFO - Job 3: Subtask download_dataset_task
[2022-08-03 04:05:37,793] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag.download_dataset_task scheduled__2022-08-02T00:00:00+00:00 [running]> on host 57fb351b7a91
[2022-08-03 04:05:37,869] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2022-08-02T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-02T00:00:00+00:00
[2022-08-03 04:05:37,870] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-03 04:05:37,871] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://storage.cloud.google.com/dtc_data_lake_de-bootcamp-357803/covid-cases.csv > /opt/***/covid-cases.csv']
[2022-08-03 04:05:37,889] {subprocess.py:85} INFO - Output:
[2022-08-03 04:05:38,014] {subprocess.py:93} INFO - Command exited with return code 0
[2022-08-03 04:05:38,070] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=download_dataset_task, execution_date=20220802T000000, start_date=20220803T040537, end_date=20220803T040538
[2022-08-03 04:05:38,154] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-08-03 04:05:38,248] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
